lib/fog/compute/google/models/instance_group_managers.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Google
+      class InstanceGroupManagers < Fog::Collection
+        model Fog::Compute::Google::InstanceGroupManager
+
+        def all(filters = {})
```

Comment:
`data = []` should be the first line in this function to properly declare scope.

Reply:
Done.